cli.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+const meow = require('meow')
+const badgen = require('badgen')
+const icons = require('badgen-icons')
+
+const flags = {
+    status: {
+        type: 'string',
+        alias: 's'
+    },
+    subject: {
+        type: 'string',
+        alias: 'j',
+        default: ''
+    },
+    color: {
+        type: 'string',
+        alias: 'c'
+    },
+    flat: {
+        type: 'boolean',
+        alias: 'f'
+    },
+    icon: {
+        type: 'string',
+        alias: 'i'
+    },
+    iconWidth: {
+        type: 'string',
+        alias: 'w'
+    }
+}
+
+const cli = meow(`
+    Usage
+        $ badgen <options>
+
+    Options
+        --status, -s       (required) Status of the badge, right part
+        --subject, -j       Subject of the badge, left part
+        --color, -c         Color of the status
+        --flat, -f          Use the flat badge style
+        --icon, -i          Icon to use
+        --icon-width, -w    Width of the icon if not square
```

Comment:
Some refinements

```
        -s, --status <text>            Status of the badge, right part (REQUIRED)
        -j, --subject <text>           Subject of the badge, left part
        -c, --color <RGB / ColorName>  Color of the status
        -f, --flat                     Use the flat badge style
        -i, --icon [<IconName>]        Icon to use
        -w, --icon-width <Number>      Width of the icon if not square
```

Reply:
I just lowercased (required) because it felt too aggressive.

Reply:
Yep, agreed